SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/GameModeReplicationTest/GameModeReplicationTest.cpp

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "GameModeReplicationTest.h"
+
+#include "Net/UnrealNetwork.h"
+
+#include "GameFramework/GameStateBase.h"
+
+AGameModeReplicationTestGameMode::AGameModeReplicationTestGameMode()
+{
+	NetCullDistanceSquared = 0;
+}
+
+void AGameModeReplicationTestGameMode::GetLifetimeReplicatedProps(TArray<FLifetimeProperty>& OutLifetimeProps) const
+{
+	DOREPLIFETIME(AGameModeReplicationTestGameMode, ReplicatedValue);
+}
+
+bool AGameModeReplicationTestGameMode::IsNetRelevantFor(const AActor* RealViewer, const AActor* ViewTarget,
```

Comment:
Why does this need to be overridden?

Reply:
I'm doing all I can to make the GameMode object not relevant; would you suggest just sticking to the 1x3 grid, and not overriding this method?

Reply:
This will likely be ignored. IsNetRelevantFor is only called from the native NetDriver. In SpatialOS, the runtime is doing this work.